src/ansys/additive/core/porosity.py

```diff
@@ -129,39 +112,39 @@ def material(self, value):
     def size_x(self):
         """Size (m) of the simulated sample in the x dimension.
 
-        Valid values are from 1e-3 to 1e-2 m (1 to 10 mm).
+        Valid values are from :obj:`MIN_SAMPLE_SIZE` to :obj:`MAX_SAMPLE_SIZE` meters.
```

Comment:
```suggestion
        Valid values are from the :obj:`MIN_SAMPLE_SIZE` value to the
        :obj:`MAX_SAMPLE_SIZE` value.
```
Was ``meters`` after the second object a mistake?